src/mappings/utils/getters.ts

```diff
@@ -219,15 +219,16 @@ export function getPlaceOfferEvent(ctx: Context): MakeOfferEvent {
   };
 }
 
-export function getWithdrawOfferEvent(ctx: Context): BaseOfferEvent {
+export function getWithdrawOfferEvent(ctx: Context): WithdrawOfferEvent {
   const event = new MarketplaceOfferWithdrawnEvent(ctx);
+
   if (event.isV55) {
     const { who: caller, class: classId, instance: instanceId } = event.asV55;
-    return { collectionId: classId.toString(), sn: instanceId.toString(), caller: addressOf(caller) };
+    return { collectionId: classId.toString(), sn: instanceId.toString(), caller: addressOf(caller), maker: addressOf(ctx.event.call?.args.maker) };
```

Comment:
any idea how to fix this line? or have something a bit cleaner

Reply:
I think this is sufficient